src/config_class.py

```diff
@@ -81,3 +85,26 @@ def __post_init__(self):
         self.paths = (
             Paths(**self.paths) if isinstance(self.paths, dict) else self.paths
         )
+
+
+@dataclass
+class PredictParams:
+    model_path: str = r"C:\Users\user\Projekty\BaldOrNot\trainings\training_name2024-10-14_23-34-02\model.keras"
+    image_dir_path: str = (
+        r"C:\Users\user\Projekty\BaldOrNot\scrapping\downloaded_images"
+    )
```

Comment:
maybe these would be better moved to Paths in config, all other paths are there

Reply:
I understand, but I think these paths relate to a completely different functionality, so it would be better if they stayed in a separate config file.